### PR TITLE
fix: add propTypes validation to each components

### PR DESCRIPTION
**Pull request by elysee15**
Fixes SC-2056

### CODE REVIEW

File: web/gds-admin-ui/src/pages/app/details/ReviewNotes/EditReviewNote.js [hunk 1]
```diff
@@ -53,4 +54,11 @@ function EditReviewNote({ note, handleCancelEditingClick, vaspId, setIsEditable
     )
 }
 
+EditReviewNote.propTypes = {
+    note: PropTypes.object.isRequired,
+    handleCancelEditingClick: PropTypes.func.isRequired,
+    vaspId: PropTypes.number.isRequired,
```
Comment (tl6kk) [inline web/gds-admin-ui/src/pages/app/details/ReviewNotes/EditReviewNote.js:60]: Just to double-check to make sure, `vaspId` here should be number or string?

File: web/gds-admin-ui/src/layouts/LeftSidebar.js [hunk 2]
```diff
@@ -7,7 +6,7 @@ import { getMenuItems } from 'helpers/menu';
 import { getDirectoryLogo, getDirectoryName, getDirectoryURL } from 'utils';
 
 
-const SideBarContent = ({ hideUserProfile }: SideBarContentProps) => {
+const SideBarContent = ({ hideUserProfile }) => {
```
Comment (tl6kk) [inline web/gds-admin-ui/src/layouts/LeftSidebar.js:9]: Is this `hideUserProfile` something to verify too?

I saw code similar to this 🔽  was modified in other files, just pin here to check and see if we are missing this one. 
```
type LeftSidebarProps = {
    hideLogo: boolean,
    hideUserProfile: boolean,
    isLight: boolean,
    isCondensed: boolean,
};
```

**Review by tl6kk — APPROVED** [review]

File: web/gds-admin-ui/src/pages/app/details/ReviewNotes/EditReviewNote.js (hunk 1 above)
Comment (elysee15) [reply web/gds-admin-ui/src/pages/app/details/ReviewNotes/EditReviewNote.js:60]: it should be string, thank you

File: web/gds-admin-ui/src/layouts/LeftSidebar.js (hunk 2 above)
Comment (elysee15) [reply web/gds-admin-ui/src/layouts/LeftSidebar.js:9]: This verification no really matter

**Review by tl6kk — APPROVED** [review]

**Pull request merged**